misc/std.py

```diff
@@ -0,0 +1,280 @@
+import os
```

Comment:
most of this is already in `formatters.py`

Reply:
Same question here, as for the navigation file.

Reply:
we got rid of std.py because it had so many different random commands in it. Most if not all of this file can be safely removed from the PR. instead of adding `pee` here, it can be added to `misc/formatters.py` instead.

Reply:
I removed this file, and broke out almost all of of the non-repetitive code into navigation.py, formatters.py, and general.py.